snprc_ehr/src/org/labkey/snprc_ehr/table/SNPRC_EHRCustomizer.java

```diff
@@ -77,46 +78,25 @@ public void customize(TableInfo table)
             doTableSpecificCustomizations((AbstractTableInfo) table);
             doCalculatedCustomizations((AbstractTableInfo) table);
             //TODO: customizeColumns((AbstractTableInfo) table);
-        }
-    }
-
-/*  //TODO: need to add investigator foreign key to multiple tables once the arc_valid_pi table has been ETL'd. tjh
-    // Are we using investigatorId or investigatorName???
-
-    private void customizeColumns(AbstractTableInfo ti)
```

Comment:
Are we not going to need this method anymore? It says TODO so unsure what its purpose was

Reply:
We don't need it, as I mentioned in my commit comment. We decided to use the PI names in the IACUC data.